src/librustdoc/html/static/main.js

```diff
@@ -415,6 +415,14 @@ function defocusSearchBar() {
                 displayHelp(true, ev);
                 break;
 
+            case "t":
+            case "T":
+                displayHelp(false, ev);
+                ev.preventDefault();
+                var themePicker = getThemePickerElement();
+                themePicker.click();
+                themePicker.focus();
```

Comment:
Are we just gonna fallthrough to the other case?

Reply:
Oh you're right! I forgot the `break;`. Good catch!

Reply:
Fixed.